.circleci/config.yml

```diff
@@ -690,3 +754,217 @@ workflows:
+          ssh-config-script: production-ssh-config.sh
+          ssh-host: shuttle.prod.internal
+          production: true
+          requires:
+            - build-and-push-production
+          filters:
+            branches:
+              only: production
+      - approve-publish-crates:
+          type: approval
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-codegen
+          path: codegen
+          requires:
+            - approve-publish-crates
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-common
+          path: common
+          requires:
+            - publish-shuttle-codegen
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-proto
+          path: proto
+          requires:
+            - publish-shuttle-common
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-service
+          path: service
+          requires:
+            - publish-shuttle-proto
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-runtime
+          path: runtime
+          requires:
+            - publish-shuttle-service
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-aws-rds
+          path: resources/aws-rds
+          requires:
+            - publish-shuttle-runtime
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-shared-db
+          path: resources/shared-db
+          requires:
+            - publish-shuttle-aws-rds
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-secrets
+          path: resources/secrets
+          requires:
+            - publish-shuttle-shared-db
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-persist
+          path: resources/persist
+          requires:
+            - publish-shuttle-secrets
+      - publish-crates:
+          name: publish-shuttle-static-folder
+          path: resources/static-folder
+          requires:
+            - publish-shuttle-persist
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-cargo-shuttle
+          path: cargo-shuttle
+          requires:
+            - publish-shuttle-static-folder
+          filters:
+            branches:
+              only: production
+      - publish-crates:
+          name: publish-shuttle-actix-web
+          path: services/shuttle-actix-web
```

Comment:
All the services can also be in parallel.

Reply:
👍  Changed to a matrix that waits on `cargo-shuttle`.